webapp/stores/emoji_store.jsx

```diff
@@ -148,8 +149,7 @@ class EmojiStore extends EventEmitter {
 
     getEmojiImageUrl(emoji) {
         if (emoji.id) {
-            // must match Client.getCustomEmojiImageUrl
-            return `/api/v3/emoji/${emoji.id}`;
+            return Client.getCustomEmojiImageUrl(emoji.id);
```

Comment:
We haven't added any UI tests so they weren't necessary at the time. For some reason, mocha-webpack throws a syntax error when importing superagent so I don't know if those would help anyway